lib/policies/base_policy.js

```diff
@@ -87,21 +87,24 @@ class BasePolicy {
     this.maxRetries = props.maxRetries
 
     /**
-     * Use zlib compression on write or batch read commands when the command
-     * buffer size is greater than 128 bytes. In addition, tell the server to
-     * compress it's response on read commands. The server response compression
-     * threshold is also 128 bytes.
+     * Milliseconds to sleep between retries.  Enter zero to skip sleep.
+     * This field is ignored when max_retries is zero.
+     * This field is also ignored in async mode.
```

Comment:
Node.js client is always using async mode so does it even make sense to include this option?

Reply:
isn't that used in underlying c-client to "how to retry"?